person.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+    super()
+  end
+  attr_accessor :name, :age
+  attr_reader :id, :rentals
```

Comment:
- [ ] I think you should use Getters and Setters for `@rentals` because of the many-to-many relationships between the rental and the person.